app/src/main/java/com/duckduckgo/app/onboarding/ui/page/DefaultBrowserPageViewModel.kt

```diff
@@ -196,6 +199,8 @@ class DefaultBrowserPageViewModel @Inject constructor(
         }
     }
 
+    private fun AppBuildConfig.isAndroid13OrAbove(): Boolean = sdkInt >= android.os.Build.VERSION_CODES.TIRAMISU
```

Comment:
nit: move this to `AppBuildConfig`, there's already an extension function there

Reply:
The `app-build-config` is a java/kotlin module and if I add the extension function there I need to bring android dependencies (android.os.*). As this code will be removed fairly soon, I decided to put the extension function where is used.